### PR TITLE
Korelov pipeline

### PR DESCRIPTION
**Pull request by korelov**

### CODE REVIEW

**Review by patriotby07 — APPROVED** [review]

**Comment by korelov** [comment]
[TC_017.006 | Pipeline> Add new Pipeline +](https://trello.com/c/ucAe7sJL/85-tc017006-pipeline-add-new-pipeline)

File: src/test/java/KorelovPipelineCreateTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
+
+public class KorelovPipelineCreateTest extends BaseTest {
+
+    @Test
+    public void testAddPipeline() {
```
Comment (VitalyVZH) [inline src/test/java/KorelovPipelineCreateTest.java:10]: Create or Add?

File: src/test/java/KorelovPipelineCreateTest.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+        final String namePipeline = "Pipeline";
+
+        getDriver().findElement(By.xpath("//a[@title='New Item']")).click();
+        getDriver().findElement(By.id("name")).sendKeys(namePipeline);
+        getDriver().findElement(By.xpath("(//div[@class='category']/ul/li)[2]")).click();
+        getDriver().findElement(By.id("ok-button")).click();
+        getDriver().findElement(By.id("yui-gen6-button")).click();
+
+        Assert.assertEquals(getDriver().findElement(By.xpath("//h1"))
```
Comment (VitalyVZH) [inline src/test/java/KorelovPipelineCreateTest.java:20]: Try assertTrue

File: src/test/java/KorelovPipelineCreateTest.java (hunk 1 above)
Comment (korelov) [reply src/test/java/KorelovPipelineCreateTest.java]: Add

File: src/test/java/KorelovPipelineCreateTest.java (hunk 2 above)
Comment (korelov) [reply src/test/java/KorelovPipelineCreateTest.java]: Assert.assertTrue(getDriver().findElement(By.xpath("//h1"))
                .getText().contains(namePipeline));

**Pull request merged**